content/news/2020-12-17-struggles.md

```diff
@@ -0,0 +1,122 @@
+title: All about my Outreachy internship with Mixxx DJ: Struggles
+authors: Aanyu Deborah Oduman
+tags: personal, outreachy, internship
+date: 2020-12-06 18:10:00
```

Comment:
Same here, add a summary if you want to use a headline to make it look better on overview pages.

```yaml
summary: ...
```